server-backend/src/NbxplorerService.ts

```diff
@@ -191,57 +304,78 @@ export class NbxplorerService implements OnApplicationBootstrap, OnApplicationSh
         this.config = config.getOrThrow('nbxplorer', { infer: true });
     }
 
-    async getBalance(xpub: string): Promise<NBXplorerBalance> {
-        const response = await (await fetch(`${this.config.baseUrl}/derivations/${xpub}/balance`)).json();
+    getUrl(cryptoCode: string = 'btc'): string {
+        if (cryptoCode === 'btc') {
+            return this.config.baseUrl;
+        }
+        return this.config.baseUrl.replace('btc', cryptoCode);
```

Comment:
suggestion: maybe would be better to change the config to a placeholder in case the url has a `btc` in it
```suggestion
        return this.config.baseUrl.replace('{cryptoCode}', cryptoCode);
```

Reply:
done